test/ok.test.ts

```diff
@@ -114,25 +114,10 @@ test('mapOr / mapOrElse', () => {
 });
 
 test('iterable', () => {
-    let i = 0;
-    for (const char of Ok('hello')) {
-        expect('hello'[i]).toBe(char);
-        expect_string(char, true);
-        i++;
-    }
-
-    i = 0;
-    for (const item of Ok([1, 2, 3])) {
-        expect([1, 2, 3][i]).toBe(item);
-        eq<number, typeof item>(true);
-        i++;
-    }
-
-    for (const item of Ok(1)) {
-        expect_never(item, true);
-
-        throw new Error('Unreachable, Err@@iterator should emit no value and return');
-    }
+    const values = Array.from(Ok('hello'));
```

Comment:
This is unused it seems? ✂️

Reply:
Oops, yes, leftover from a previous iteration, removing.

Reply:
```suggestion
```